utils/renderEslint.ts

```diff
@@ -35,6 +35,11 @@ export default function renderEslint(
     additionalDependencies['eslint-plugin-cypress'] = eslintDeps['eslint-plugin-cypress']
   }
 
+  if (needsPlaywright) {
+    additionalConfig.extends = ['plugin:playwright/recommended']
```

Comment:
shouldn't the plugin apply only the Playwright files (as done for Cypress above)?

Reply:
Yes, ESLint will apply the rules defined in the root of your configuration to all files in your project (which is not ideal). 
@georgi-mateev Can you target **e2e** folder - something like `files: ['e2e/**/*.{js,ts,jsx,tsx}']`?